dlf/plugins/toolbox/tools/imagedownload/class.tx_dlf_toolsImagedownload.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * (c) Kitodo. Key to digital objects e.V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+/**
+ * Tool 'Image Download' for the plugin 'DLF: Toolbox' of the 'dlf' extension.
+ *
+ * @author	Alexander Bigga <alexander.bigga@slub-dresden.de>
+ * @package	TYPO3
+ * @subpackage	tx_dlf
+ * @access	public
+ */
+class tx_dlf_toolsImagedownload extends tx_dlf_plugin {
+
+	public $scriptRelPath = 'plugins/toolbox/tools/pdf/class.tx_dlf_toolsImagedownload.php';
```

Comment:
There is a typo in the path. It should be "plugins/toolbox/tools/**imagedownload**/" instead of "plugins/toolbox/tools/**pdf**/".